src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.fuse.apicurio.jaxrs.tests;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.redhat.fuse.apicurio.jaxrs.GenerateFuseProjectResource;
+
+import io.swagger.models.Swagger;
+import io.swagger.parser.SwaggerParser;
+import io.swagger.util.Json;
+
+public class GenerateFuseProjectResourceTest {
+
+	/**
+	 * Not the cleanest approach, but uses the same approach as the generator to
+	 * create the Rest DSL configuration and return it as a string.
+	 * 
+	 * @param filePath
+	 * @return
+	 * @throws Exception
+	 */
+	private String getRestDSLXMLFromSwagger(String filePath) throws Exception {
+		GenerateFuseProjectResource generator = new GenerateFuseProjectResource();
+		File swaggerFile = new File(filePath);
+		Assert.assertTrue(swaggerFile.exists());
```

Comment:
@apupier - Though the assertj idea is a good one, we'll stick with this for now and shift in a separate PR as we add more tests. But I'm not sure what you meant by handling this through a static import? Do you have an example?

Reply:
https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html